artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/AMQPBrokerConnection.java

```diff
@@ -498,7 +498,7 @@ private Queue installMirrorController(AMQPMirrorBrokerConnectionElement replicaC
       Queue mirrorControlQueue = server.locateQueue(getMirrorSNF(replicaConfig));
 
       if (mirrorControlQueue == null) {
-         mirrorControlQueue = server.createQueue(new QueueConfiguration(getMirrorSNF(replicaConfig)).setAddress(getMirrorSNF(replicaConfig)).setRoutingType(RoutingType.ANYCAST).setDurable(replicaConfig.isDurable()).setInternal(true), true);
+         mirrorControlQueue = server.createQueue(new QueueConfiguration(getMirrorSNF(replicaConfig)).setAddress(getMirrorSNF(replicaConfig)).setRoutingType(RoutingType.ANYCAST).setDurable(replicaConfig.isDurable()).setInternal(false), true);
```

Comment:
@gemmellr I think it was a mistake to hide management. There is some movement to make internal queues managed. I'm not sure we merged that one...

Like the test I wrote for example, I need to manage from the queue.


So is the user I'm dealing with now.. they need to manage it.. check sizes, do operations.. etc...

Reply:
I have no concern around whether it can be managed or not. I'm saying that my impression from the prior change that added the 'dual mirror' stuff and made it 'internal' was that it was done more to have it treated a specific way (e.g excluded) by other e.g mirror/cluster/etc related code in the broker and not just to hide the queue from management. I am raising the question over whether thats actually the case and if so whether any previously desired treatment elsewhere wont occur as a result from simply marking it not-internal again.

This didnt occur to me before, but while typing that it did... I believe Justin (or maybe you?) made a change to persist the 'internal' value at some point...if so is the broker code actually going to end up dealing with a mixture now where some are and some arent?

Reply:
I am keeping it as internal, but allowing managemen ton this queue, by calling the register directly. I also open a new JIRA for this.